influxql/call_iterator_test.go

```diff
@@ -731,6 +871,60 @@ func benchmarkDistinctIterator(b *testing.B, pointN int) {
 	}
 }
 
+func BenchmarkMedianIterator_1K(b *testing.B)   { benchmarkMedianIterator(b, 1000) }
```

Comment:
I really like the benchmarks for the median iterator, but is it possible for you to split this work into a separate commit and a separate PR? I would like to keep this PR as "only the implementation of `mode()`".